Update README

Thielak opened the pull request:
- Documented a couple of new optional flags
- Added a link to a third party repository of embeddings
- Reworded a few parts
- Fixed some typos

Thielak committed:
Update README

- Documented a couple of new optional flags
- Added a link to a third party repository of embeddings
- Reworded a few parts
- Fixed some typos

AUTOMATIC1111 commented:
token_identifier.txt should be token_identifier.bin

Auth comamnd line option is different now, it's --gradio-auth

Thielak committed:
Updated README

- small update to reflect the revised commit for gradio authentication

Thielak commented:
I updated the readme to reflect the change to the authentication flag.
As for the token_identifier.txt part, it's a file that goes alongside embeddings files on the linked repo (e.g. https://huggingface.co/sd-concepts-library/pantone-milk/tree/main), I wasn't suggesting to download the .bin file and renaming it to that. I'm open to suggestions on how to make that part clearer if you feel like it'd be misleading

AUTOMATIC1111 commented:
Oh. You can rename embedding file to anything you want. You can download a .bin trained with token <pantone-milk>, name it Clown.bin, and then use the word Clown in the prompt to get that embedding. If you want to write about the txt files, mention that it is a suggestion; now it looks more like a requirement.

Thielak committed:
Update README.md

Revision of the previous commit to make it less misleading and mention the option of using .bin files

Thielak commented:
Good call, as it's not required I left it out completely to reduce possible misunderstandings

AUTOMATIC1111 commented:
Thanks.
